data/templates/base/%(KEYBOARD)s.c

```diff
@@ -0,0 +1,4 @@
+// Copyright %(YEAR)s %(YOUR_NAME)s (@%(USER_NAME)s)
```

Comment:
I'm not sure how I feel about % and () in a filename, makes for awkward shell.

Reply:
Yeah I agree. Left it in as I figured it'd be discussed then.